from __future__ import annotations

class OutOfBoundsDatetime(ValueError): ...
